ExportKindleClippingsToNotion.Tests/ClippingsExtractorTests.cs

```diff
@@ -0,0 +1,108 @@
+using FluentAssertions;
+
+namespace ExportKindleClippingsToNotion.Tests;
+
+public class ClippingsExtractorTests
+{
+    [Theory]
```

Comment:
Was macht Theory?

Reply:
Markiert die Methode als Testmethode, welche mit verschiedenen Parametern ausgeführt wird.

Diese Methode wird also einmal pro InlineData Attribut ausgeführt. Der Wert aus dem Attribut wird in die Methode gereicht.

Das ist auch abhängig vom TestFramework. Ich habe hier xUnit eingebaut.